models/experimental/functional_mistral/tt/ttnn_functional_attention.py

```diff
@@ -15,21 +14,21 @@ def apply_rotary_emb(xq, xk, bcast_freq_xq, bcast_freq_xk, device, mem_config):
     xq_real = torch_to_tt_tensor_rm(t_xq[..., :, :, ::2], device)
     xq_img = torch_to_tt_tensor_rm(t_xq[..., :, :, 1::2], device)
 
-    xq = tt_lib.tensor.complex_tensor(xq_real, xq_img)
+    xq = ttnn.complex_tensor(xq_real, xq_img)
 
     xq_real.deallocate()
     xq_img.deallocate()
 
     xk_real = torch_to_tt_tensor_rm(t_xk[..., :, :, ::2], device)
     xk_img = torch_to_tt_tensor_rm(t_xk[..., :, :, 1::2], device)
-    xk = tt_lib.tensor.complex_tensor(xk_real, xk_img)
+    xk = tttnn.complex_tensor(xk_real, xk_img)
 
     xk_real.deallocate()
     xk_img.deallocate()
 
-    xq_out = tt_lib.tensor.complex_mul(xq, bcast_freq_xq, output_mem_config=mem_config)
+    xq_out = ttnn.mul(xq, bcast_freq_xq, output_mem_config=mem_config)
 
-    xk_out = tt_lib.tensor.complex_mul(xk, bcast_freq_xk, output_mem_config=mem_config)
+    xk_out = ttnn.mul(xk, bcast_freq_xk, output_mem_config=mem_config)
```

Comment:
memory_config= kw ???

Reply:
till now there is no equivalent op for this mug in main